client/components/gsuite/gsuite-features/index.jsx

```diff
@@ -19,29 +15,25 @@ const GSuiteFeatures = ( { compact, domainName, productSlug, type } ) => {
 	const getStorageText = () => {
 		if ( compact ) {
 			return undefined;
-		} else if (
-			[ GOOGLE_WORKSPACE_BUSINESS_STARTER_YEARLY, GSUITE_BASIC_SLUG ].includes( productSlug )
-		) {
-			return translate( 'Get 30GB of storage for all your files synced across devices.' );
-		} else if ( GSUITE_BUSINESS_SLUG === productSlug ) {
+		}
+
+		if ( GSUITE_BUSINESS_SLUG === productSlug ) {
```

Comment:
I like these restructures! I am also wondering whether we can push this into a separate PR, as it would be really easy and quick to test on its own, and from a quick look it seems like it doesn't have any dependencies on the rest of this PR.